app/src/main/java/org/mozilla/tv/firefox/pocket/PocketVideoFetchScheduler.kt

```diff
@@ -111,9 +111,21 @@ class PocketVideoFetchScheduler(
         return userFetchTime.timeInMillis - now.timeInMillis
     }
 
-    fun setDelayForQA(seconds: Long) {
-        IS_QA_BUILD = true
-        DELAY_TIME_FOR_QA_MILLIS = TimeUnit.SECONDS.toMillis(seconds)
+    /**
+     * Used for QA to set the fetch delay to a specified number of seconds. This is set through
+     * a custom intent.
+     */
+    fun setQAFetchDelayOverride(seconds: Long) {
+        isQABuild = true
+        qaFetchDelayOverrideMillis = TimeUnit.SECONDS.toMillis(seconds)
+    }
+
+    /**
+     * Used to reset QA fetch delay values after each use so we do not end up spamming the Pocket servers.
```

Comment:
The first sentence of a function should always be what the function does, not how it's intended to be used. For example, "Resets fetch delay to production values. We do this so we do not spam the Pocket servers..." fwiw, sometimes it's hard to make this different from the method name but if you can, it's usually worth it.

The reasons for saying what the method does in the first sentence is:
- iirc, generated javadoc will only use the first sentence in the summary views (i.e. the list of all the methods; not sure what kdoc does)
- Devs won't necessarily read all of your kdoc so you should tell them ASAP what effects it will have on the program (does it have side effects? Does it just return values?)

And we generally want to avoid saying _how_ it's being used because the code can change and that comment can be set out of date. However, it's valuable to say _why_ it might be used (e.g. so we don't spam the Pocket servers) because that may affect the caller's decision to use it.

I hope this makes sense and is actually correct – I'm a little tired. :P

Here and above.